kotlin-android/app/src/main/java/com/mongodb/tasktracker/MemberActivity.kt

```diff
@@ -0,0 +1,116 @@
+package com.mongodb.tasktracker
+
+import android.app.AlertDialog
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.widget.EditText
+import android.widget.Toast
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import com.mongodb.tasktracker.model.*
+import io.realm.Realm
+import io.realm.mongodb.functions.Functions
+import org.bson.Document
+import java.util.*
+
+class MemberActivity : AppCompatActivity() {
+    private var user: io.realm.mongodb.User? = null
+    private lateinit var realm: Realm
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var adapter: MemberAdapter
+    private lateinit var fab: FloatingActionButton
+    private lateinit var members: ArrayList<Member>
+
+    override fun onStart() {
+        super.onStart()
+        try {
+            user = taskApp.currentUser()
+        } catch (e: IllegalStateException) {
+            Log.w(TAG(), e)
```

Comment:
Would it be helpful to add some explanatory text about the error similar to what you've done on lines 75 and 111? I'd recommend this for all instances of runnable code designed to teach.

Reply:
In all honesty I'm not quite sure how necessary this is. I think it's there since this is a network request and we need to handle situations where there is no network connection gracefully, but I've only included it because it's a pattern I've seen Christian use before in his java/kotlin apps and I treat it as a best practice. Reaching out about this to make sure I know if/why it's really needed.

Reply:
Removed these exception handlers -- confirmed that they are not necessary!